packages/docusaurus-plugin-openapi/src/markdown/createParamsTable.ts

```diff
@@ -70,6 +70,12 @@ export function createParamsTable({ parameters, type }: Props) {
                     children: createDescription(message),
                   })
                 ),
+                guard(param.schema?.description, (description) =>
```

Comment:
Will this show 2 descriptions if specified in both places? Is that the expected behavior? Or should we modify the description below to be `guard(param.description ?? param.schema?.description, (description) =>`

Reply:
Not sure what the correct behavior should be. I was thinking something similar, but then I thought that one may want to use the schema description to specify information about the values themselves (like, "string: valid strings should start with a +"), and the parameter description to specify info about the param itself (like, "a phone number")

Reply:
Cool let’s keep both then :)